src/cloud.c

```diff
@@ -56,60 +56,83 @@
 #define AMQP_CMD_DATA_PUBLISH "data.publish"
 #define AMQP_CMD_DEVICE_UNREGISTER "device.unregister"
 
-struct cloud_callbacks {
+/* Hashmap to user callback from southbound traffic */
+static struct l_hashmap *map_event_to_cb;
+
+/* Union with all cloud callbacks */
+union cloud_cb_t {
 	cloud_downstream_cb_t update_cb;
-	cloud_downstream_cb_t request_cb;
```

Comment:
Why did you remove this callback from the union?

Reply:
Done